report_thesis/src/sections/baseline.tex

```diff
@@ -0,0 +1,63 @@
+\section{Baseline}
+In \citet{p9_paper}, we presented our efforts to replicate the MOC model by \citet{cleggRecalibrationMarsScience2017}.
+This effort was motivated by our desire to better understand the model and its performance, and to experiment with its components in order to determine how it could be improved.
+However, as discussed in that paper, there were some differences between our replica and the original model.
+These differences were caused by missing information in the original paper, and so rather than introducing our own assumptions, we designed experiments to determine the best way to replicate the model.
+
+Initially, our replication omitted the use of the \gls{mad} for outlier removal in the data preprocessing stage, a step included in the original model.
+The omission was due to the lack of information regarding its implementation within the processing pipeline.
+To rectify this, we experimented with applying \gls{mad} at various stages in the pipeline.
+We determined that its application post-removal of the initial five shots, and prior to masking and normalization, most closely mirrored the outcomes of the original model.
+
+Furthermore, our replica only utilized a single dataset for the \gls{ica} phase, while the original model used all five datasets.
+This difference was due to the original paper not specifying how the five datasets were used, and so we designed an experiment to determine how to use them in a way that would most closely replicate the original model.
```

Comment:
Also dont really feel like the first half about what the original paper did not do. The second half is good.